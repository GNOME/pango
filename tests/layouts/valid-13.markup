wrap=word,tabs=0 50 100 150 200,single_paragraph=true
a       b       c       dâ€¨e     f       g       h
