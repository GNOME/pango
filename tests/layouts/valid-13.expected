a       b       c       d e     f       g       h

--- parameters

wrapped: 0
ellipsized: 0
lines: 1

--- attributes

range 0 2147483647

--- directions

0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 

--- cursor positions

0(0) 1(0) 2(0) 3(0) 4(0) 5(0) 6(0) 7(0) 8(0) 9(0) 10(0) 11(0) 12(0) 13(0) 14(0) 15(0) 16(0) 17(0) 18(0) 19(0) 20(0) 21(0) 22(0) 23(0) 24(0) 25(0) 28(0) 29(0) 30(0) 31(0) 32(0) 33(0) 34(0) 35(0) 36(0) 37(0) 38(0) 39(0) 40(0) 41(0) 42(0) 43(0) 44(0) 45(0) 46(0) 47(0) 48(0) 49(0) 50(0) 51(0) 51(1) 

--- lines

i=1, index=0, paragraph-start=1, dir=ltr 'a       b       c       d e     f       g       h
'

--- runs

i=1, index=0, chars=25, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'a       b       c       d'
[0,-1]show=2
i=2, index=25, chars=1, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, ' '
[0,-1]show=2
i=3, index=28, chars=24, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'e     f       g       h
'
[0,-1]show=2
i=4, index=52, no run, line end
